Packages/Tracking/Core/Runtime/Scripts/XRHandsSubsystem/SubsystemStarter.cs

```diff
@@ -92,13 +92,35 @@ private static void RunAfterSceneLoad()
 
             LeapProvider leapProvider = Hands.Provider;
 
+            // If there is no leap provider in the scene
             if (leapProvider == null)
             {
                 Debug.Log("There are no Leap Providers in the scene, automatically assigning one for use with Leap XRHands");
                 leapProviderGO = new GameObject("LeapXRServiceProvider");
-                leapProvider = (LeapProvider)leapProviderGO.AddComponent<LeapXRServiceProvider>();
+                LeapXRServiceProvider leapXRServiceProvider = leapProviderGO.AddComponent<LeapXRServiceProvider>();
+                leapXRServiceProvider.PositionDeviceRelativeToMainCamera = false;
+                leapProvider = (LeapProvider)leapXRServiceProvider;
                 GameObject.DontDestroyOnLoad(leapProviderGO);
             }
+            else
+            {
+                Debug.LogWarning("We recommend that you do not add a Leap Provider to scenes using the Leap Subsystem for XR Hands. This can cause unwanted behaviour.");
+            }
```

Comment:
I think we should still _make_ users a LeapProvider here - but warn them about the behaviour - that way, the positioning issue will be avoided even after warning them

Reply:
This might cause conflicting leap providers in scene, unless we specifically use this leap provider for the subsystem and not other hands in the scene?

Reply:
You can have multiple and it be fine, and we will create one specifically referenced for the Subsystem - if there's already one in the scene, it should already be referenced for the classes that use it - or they should be using the ProviderManager which will be autodetected before a standard LeapXRServiceProvider when using the static provider utilities